Tribler/Main/vwxGUI/home.py

```diff
@@ -12,6 +13,8 @@
 import binascii
 from time import strftime, time
 from traceback import print_exc
+from binascii import hexlify
```

Comment:
There are three pylint warnings because of the import order in this file. Easy to fix with your IDE ;)